### DISCUSSION TITLE
Scrobbling support

### DISCUSSION THREAD

**Discussion by aade-sh** [jpochyla/psst#460] [General]
Hey maintainers,

I recently started using psst and loving it so far. I am an obsessive [scrobbler](https://www.dictionary.com/browse/scrobble) and unfortunately it turned out Psst doesn't yet support it. 

I want to contribute to the project and implement this functionality, which, I believe, is related to one of the items in the roadmap titled "Reporting played tracks to Spotify servers".

Could anyone point me in the right direction for getting this started? Just need to know which files or parts of the project I should dive into to get this going.

**Comment by viridivn** [comment]
[rescrobbled](https://github.com/InputUsername/rescrobbled) works by listening to the D-Bus directly reporting to last.fm, if that's the scrobbling service you use

**Comment by Pogodaanton** [comment]
Last.fm support was introduced with #594
